Esri/arcgis-maps-sdk-swift-samples
# [Fix] Work around map view keyboard jump bug

## Description

This PR implements a workaround for a bug where presenting a keyboard in landscape mode would cause a map view in a sample to jump to the bottom of the map.

## How To Test

- Present a keyboard in landscape mode while in a sample with a map. Example: 
    - Go to `All` -> `Set Basemap`.
    - Tap the "All" category search bar to present the keyboard (SHIFT-CMD-K if it doesn’t in simulator).
    - Ensure the viewpoint of the map view stays the same.
